.github/workflows/operator-integration-tests-v2.yml

```diff
@@ -293,7 +311,8 @@ jobs:
           cat ~/.jina/config.json
       - name: Run tests
         run: |
-          pytest -v tests/integration/test_end_to_end.py
+          focus_files=$(echo ${{ matrix.test-path }} | xargs -n1 | xargs -I{} echo "--focus-file tests/integration/remote{}" | xargs)
```

Comment:
This seems to be copied from operator/api tests. `--focus-file` is an arg passed to `ginkgo` & is unrelated to `pytest`.